### PR TITLE
design: Rev routing design doc

### PR DESCRIPTION
**Pull request by stevesloka**
Signed-off-by: Steve Sloka <slokas@vmware.com>

### CODE REVIEW

File: design/routing-design.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 _Status_: Draft
 
-The IngressRoute design introduced in July 2018 the design chose to only support prefix matching for routes. This was mostly a time-to-market decision, but also reflected the fact that we didn't know other ways customers might want to match routes. We knew that Envoy also supported other methods, but without a signal from our user base, blindly adding support for all of the existing mechanisms felt like throwing the problem over the wall.
+The HTTPProxy design introduced in July 2018 the design chose to only support prefix matching for routes. This was mostly a time-to-market decision, but also reflected the fact that we didn't know other ways customers might want to match routes. We knew that Envoy also supported other methods, but without a signal from our user base, blindly adding support for all of the existing mechanisms felt like throwing the problem over the wall.
```
Comment (davecheney) [inline design/routing-design.md]: suggestion: maybe this should stay as IngressRoute

File: design/routing-design.md [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
 
 It's clear today that only supporting prefix routing is too limiting. Customers want to route not just on prefix, substring, and regex -- the three we identified last year -- but also header matching, source ip, user agent, and many more.
 
-The scope of this design doc looks to improve on the IngressRoute.v1beta1 design and plan how the current design will change to support these additional routing features, yet still preserve the current multi-team capabilities through delegation.
+The scope of this design doc looks to improve on the HTTPProxy.v1beta1 design and plan how the current design will change to support these additional routing features, yet still preserve the current multi-team capabilities through delegation.
```
Comment (davecheney) [inline design/routing-design.md]: s/HTTPProxy.v1beta1/the IngressRoute design.

File: design/routing-design.md (hunk 1 above)
Comment (stevesloka) [reply design/routing-design.md]: ahh yup, bitten by the find/replace. =)

**Comment by stevesloka** [comment]
@davecheney I revved this doc with the changes from #1501, but want to agree on that first.

**Pull request merged**